clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiConverter.groovy

```diff
@@ -977,15 +977,19 @@ class KubernetesApiConverter {
     return hasDeployment(description) ? [(parsedName.cluster): "true"] : [(name): "true"]
   }
 
+  static Map<String, String> restrictedServerGroupLabels(String name) {
+    return restrictedServerGroupLabels(name, false)
+  }
+
   /*
    * This represents the set of labels that differentiate replica sets from deployments - these are needed so
    * different replica sets under the same deployment don't apply to the same pods
    */
-  static Map<String, String> restrictedServerGroupLabels(String name) {
+  static Map<String, String> restrictedServerGroupLabels(String name, boolean requireRCLabel) {
```

Comment:
This param name is much better, but I'm a little confused with what the intended model is now. In other places the stateful & daemon sets aren't being treated as "server groups", e.g. caching, but here you apply labels made for server groups with special rules around what labels get applied. Maybe this should be done in separate non-server group function if that's the intent?

Reply:
I just don't feel that RC label name applicable to statefulset.  This is why I try to skip.  On the other hand, I would like to keep other labels because we don't have any custom labels now.  I can skip this implementation. Suggestions?